src/components/EventDetails/EventDetails.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+    Typography,
+    Grid,
+    Button,
+    Avatar,
+    Fab,
+    Chip,
+} from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
+import { Stack, ImageList, ImageListItem } from '@mui/material';
+import { imageItems } from '../../contexts/UserContext';
+import { EventInfoPageProps } from '../../pages/EventInfo/EventInfo';
+import { eventInfoStyles } from '../../pages/EventInfo/EventInfoStyle';
+import { getDatePart } from '../../tools/index.js'
+
+export interface EventDetailsProps {
+    eventDetails: EventInfoPageProps, 
+    isSubscribed: boolean, 
+    subscribedValidationHandler: () => void
+  }
+
+
+const EventDetails: React.FC<EventDetailsProps> =({ eventDetails, isSubscribed, subscribedValidationHandler}) => {
+    const classes = eventInfoStyles()
+    return (
+        <>
+            <Grid className={classes.titleContainer}>
+                <Grid xs={12} sm={7} className={classes.eventTitleRow}>
+                    <Typography variant="h4" component="div" aria-label='title'>
+                        {eventDetails?.name}
```

Comment:
DRY, dont repeat yourself
Please destructure eventDetails so you can use the properties without repeating the eventDetails prop

const { name } = eventDetails;
```suggestion
                        {name}
```